src/pages/validation-rules-analysis/validation-rules-analysis-table/ValidationRulesAnalysisTable.test.js

```diff
@@ -77,4 +121,13 @@ describe('Test <ValidationRulesAnalysisTable /> rendering:', () => {
     it('Render "DownloadAs" component.', () => {
         expect(wrapper.find(DownloadAs).length).toBe(2)
     })
+
+    it('Should render attribute option combo column if not default', () => {
+        expect(
+            withAttrOptCombos()
+                .find(TableRow)
+                .at(1)
+                .find(TableRowColumn).length
+        ).toBe(9) // First row after header
```

Comment:
I don't mind this too much, but it's quite brittle. I think checking if the header cell label is existing would be more resilient to change. But it does work for now, so I don't care too much

Reply:
Agreed, I just copied the old test. Will look into checking the header.